### DISCUSSION TITLE
Add support for reopening autoclosed changes with Gerrit

### DISCUSSION THREAD

**Discussion by felipecrs** [renovatebot/renovate#28773] [Suggest an Idea]
### Tell us more.

GitHub relevant code: https://github.com/renovatebot/renovate/blob/8ce089bacb0fa9d2ee4258e47cea0178535607b3/lib/modules/platform/github/index.ts#L943-L996

Gerrit relevant code: https://github.com/renovatebot/renovate/blob/8ce089bacb0fa9d2ee4258e47cea0178535607b3/lib/modules/platform/gerrit/index.ts#L220

This more like a reminder for myself, I may work on this.

@NiasSt90 do you see any impediments in this?

**Comment by rarkins** [comment]
Is this a blocker, or nice to have? e.g. what is the behavior today - do autoclosed changes block the updates, or are updates created in a new branch/PR?

For the referenced github, this was an optimization rather than essential feature.

I can convert this to an issue once there's agreement on the above

**Reply by felipecrs** [reply]
To be honest with you I have no idea what real improvement this would bring. Certainly, it's not a blocker.

However, Gerrit does have APIs to reopen closed changes, so theoretically this could be implemented.

I was just investigating the code base to try to fix https://github.com/renovatebot/renovate/pull/28774 and noticed this difference between the GitHub provider and Gerrit, so I thought this was a missing feature that should be tracked.

If you believe this has no importance, feel free to close this ticket.

**Reply by rarkins** [reply]
It's an optimization but definitely not necessary. So you're welcome to submit a PR if you like

**Reply by felipecrs** [reply]
Sounds good. Thank you.
